applications/ShallowWaterApplication/custom_conditions/wave_condition.cpp

```diff
@@ -181,6 +181,11 @@ void WaveCondition<TNumNodes>::InitializeData(
     ConditionData& rData,
     const ProcessInfo& rProcessInfo)
 {
+    if (rProcessInfo.Has(INTEGRATE_BY_PARTS)) {
+        rData.integrate_by_parts = rProcessInfo[INTEGRATE_BY_PARTS];
+    } else {
+        rData.integrate_by_parts = false;
+    }
```

Comment:
```suggestion
    rData.integrate_by_parts = rProcessInfo[INTEGRATE_BY_PARTS];
```

since it is passed as `const` it will return `false` if it doesn't have `INTEGRATE_BY_PARTS`

Reply:
thanks for the hint!